src/Formlet/Ocelot/CheckboxSet.purs

```diff
@@ -0,0 +1,158 @@
+module Formlet.Ocelot.CheckboxSet
+  ( Render(..)
+  , checkboxSet
+  , enumCheckboxSet
+  , genericCheckboxSet
+  ) where
+
+import CitizenNet.Prelude
+
+import Data.Set as Data.Set
+import Formlet as Formlet
+import Formlet.Ocelot.Enum as Formlet.Ocelot.Enum
+import Formlet.Render as Formlet.Render
+import Option as Option
+import Record as Record
+
+type Form config m options render a =
+  Formlet.Form
+    { readonly :: Boolean | config }
+    (Formlet.Render.Render options (checkboxSet :: Render | render))
+    m
+    (Set a)
+    (Set a)
+
+newtype Render a =
+  Render
+    { columns :: Int -- NOTE positive integer (> 0)
```

Comment:
Maybe not worth the effort, but we could use a UInt for this: https://pursuit.purescript.org/packages/purescript-uint/7.0.0

Reply:
Since we're probably the only consumers of the library at the moment, I'd wait until either we keep making mistakes or someone else opens an issue.